### DISCUSSION TITLE
Audio-I2S drop down box also includes time zone selections

### DISCUSSION THREAD

**Discussion by Sagnasty** [rern/rAudio#1304] [Fixed Issues]
Running rAudio 20230310

![image](https://github.com/rern/rAudio/assets/76575081/425b1e55-f561-4808-b233-303fde0205e1)

3 x Raspberry Pi 3B+ Rev 1.3 > HiFiBerry DAC+ Pro
1 x Raspberry Pi 3B Rev 1.2 > HiFiBerry DAC+ Pro
Synology DS718+

Jim

**Comment by klassikmann** [comment]
Same here (rAudio 20230522)

**Reply by Sagnasty** [reply]
Correction: I am running rAudio 20230522

**Reply by ceejayemm** [reply]
Same here with 20230522 on RPi3A and RPi3B+

**Comment by klassikmann** [comment]
rAudio 20230523: Fixed
